### ISSUE TITLE
String extraction for Transifex cannot handle linebreaks and tabs

### ISSUE THREAD

**Issue by raimund-schluessler** [nextcloud/server#22037]
With https://github.com/nextcloud/tasks/pull/1124 we put another argument in the argument list of the Nextcloud translate function `t` and we made the code better readable by inserting line breaks like so:

Before:
```
check.msg = this.$t('tasks', 'The name "{calendar}" is already used.', { calendar: name })
```

After:
```
check.msg = this.$t(
    'tasks',
    'The name "{calendar}" is already used.',
    { calendar: name },
    undefined,
    { sanitize: false, escape: false }
)
```

After merging the pull request, all strings to translate from the lines affected by this change were removed from Transifex, see https://github.com/nextcloud/tasks/commit/bfc245fa2052a03ddc5a168d87f2682f69b1e948. I haven't figured out what cause the issue, if it is the linebreak of the other argument.

I don't really know who to ask to check whether the string extraction from Transifex works correctly and can handle linebreaks, so I am gonna ping @ChristophWurst, since you worked on the translation in https://github.com/nextcloud/server/pull/20976 🙈 

I will check whether removing the linebreaks might solve the issue.

**Comment by ChristophWurst** [comment]
Yeah, sorry, this logic is a bit fragile. We've all run into this at some point or another :wink:

**Comment by szaimen** [comment]
I suppose this issue is still valid?

**Comment by raimund-schluessler** [comment]
> 
> 
> I suppose this issue is still valid?

Yes.

**Comment by szaimen** [comment]
Hi, please update to 24.0.9 or better 25.0.3 and report back if it fixes the issue. Thank you!

My goal is to add a label like e.g. 25-feedback to this ticket of an up-to-date major Nextcloud version where the bug could be reproduced. However this is not going to work without your help. So thanks for all your effort!

If you don't manage to reproduce the issue in time and the issue gets closed but you can reproduce the issue afterwards, feel free to create a new bug report with up-to-date information by following this link: https://github.com/nextcloud/server/issues/new?assignees=&labels=bug%2C0.+Needs+triage&template=BUG_REPORT.yml&title=%5BBug%5D%3A+

**Comment by raimund-schluessler** [comment]
I suspect that the issue still persists, but I didn't check it, since it would require deliberately (possibly) breaking the string extraction for e.g. the Tasks app.

**Comment by szaimen** [comment]
Fine by me to reopen
